services/maven-central/maven-central-snapshots-last-update.service.js

```diff
@@ -0,0 +1,51 @@
+import Joi from 'joi'
+import { pathParams } from '../index.js'
+import { parseDate, renderDateBadge } from '../date.js'
+import { nonNegativeInteger } from '../validators.js'
+import MavenCentralSnapshotsBase from './maven-central-snapshots-base.js'
+
+const updateResponseSchema = Joi.object({
+  metadata: Joi.object({
+    versioning: Joi.object({
+      lastUpdated: nonNegativeInteger,
+    }).required(),
+  }).required(),
+}).required()
+
+export default class MavenCentralSnapshotsLastUpdate extends MavenCentralSnapshotsBase {
+  static category = 'activity'
+
+  static route = {
+    base: 'maven-central-snapshots/last-update',
+    pattern: ':groupId/:artifactId',
+  }
+
+  static openApi = {
+    '/maven-central-snapshots/last-update/{groupId}/{artifactId}': {
+      get: {
+        summary: 'Maven Central Snapshots Last Update',
+        parameters: pathParams(
+          { name: 'groupId', example: 'com.google.guava' },
+          { name: 'artifactId', example: 'guava' },
```

Comment:
Can we replace this with an example that works

Reply:
OK, sure. But if you look at the example used in the service tests, it does actually exist
https://central.sonatype.com/repository/maven-snapshots/me/mrdoc/minecraft/dlibcustomextensions/maven-metadata.xml
the versions might expire over time, but the repo is there
https://central.sonatype.com/repository/maven-snapshots/com/google/guava/guava/maven-metadata.xml
is never going to return anything.